Tests/feaLib/builder_test.py

```diff
@@ -154,6 +156,7 @@ def check_fea2fea_file(self, name, base=None, parser=Parser):
                 sys.stderr.write(line+"\n")
             self.fail("Fea2Fea output is different from expected. "
                       "Generated:\n{}\n".format("\n".join(actual)))
+        parser.ignore_comments = temp
```

Comment:
a `try: ... finally: ...` block will _not_ capture exceptions but let them propagate; it will only ensure that `finally:` statement is always called whether exceptions are raised or not. If one wishes to handle an exception, one needs to use `except:`.

the temporary variable is risky because if anything goes wrong (the test errors or fails) it will change the global state of the module.